framework/PostProcessorFunctions/riskMeasuresDiscrete.py

```diff
@@ -222,3 +351,60 @@ def run(self,inputDic):
     # outputDic['data']['input'] = {} # {'dummy' : np.asanyarray(0)}
 
     return outputDic
+
+  def runDynamic(self,inputDic,timeHistory):
+    """
+     This method perform the dynamic calculation of the risk measures
+     @ In, inputDic, list, list of dictionaries which contains the data inside the input DataObjects
+     @ In, timeHistory, dict, dictionary containing  boolean temporal profiles (0 or 1) of a sub set of the input variables
+     @ Out, outputDic, dict, dictionary which contains the risk measures
+    """
+    # timeHistory format values:
+    # - 0 component is disconnected
+    # - 1 component is connected
+
+    if self.temporalID is None:
+      self.raiseAnError(IOError, 'RiskMeasuresDiscrete Interfaced Post-Processor: an HistorySet is provided but no temporalID variable is specified')
+
+    if self.temporalID not in timeHistory['data']['output'][1].keys():
```

Comment:
Again, I just want to get a sense of what this 1 is looking at? Maybe a comment in the code to specify what this 1 is doing could be useful?